ogr/services/pagure.py

```diff
@@ -317,6 +317,43 @@ def get_branches(self) -> List[str]:
     def get_description(self) -> str:
         return self.get_project_info()["description"]
 
+    def get_issue_list(self, status: IssueStatus = IssueStatus.open) -> List[Issue]:
+        payload = {"status": status.name.capitalize()}
+
+        raw_issues = self._call_project_api("issues", params=payload)["issues"]
+        issues = [self._issue_from_pagure_dict(issue_dict) for issue_dict in raw_issues]
+        return issues
+
+    def get_issue_info(self, issue_id: int) -> Issue:
+        raw_issue = self._call_project_api("issue", str(issue_id))
+        return self._issue_from_pagure_dict(raw_issue)
+
+    def _get_all_issue_comments(self, issue_id: int) -> List[IssueComment]:
+        raw_comments = self._call_project_api("issue", str(issue_id))["comments"]
+        return [
+            self._issuecomment_from_pagure_dict(raw_comment)
+            for raw_comment in raw_comments
+        ]
+
+    def issue_comment(self, issue_id: int, body: str) -> IssueComment:
+        payload = {"comment": body}
+        self._call_project_api("issue", str(issue_id), "comment", data=payload, method="POST")
+        return IssueComment(comment=body, author=self._username)
+
+    def create_issue(self, title: str, body: str) -> Issue:
+        payload = {
+            "title": title,
+            "issue_content": body
+        }
+        new_issue = self._call_project_api("new_issue", data=payload, method="POST")["issue"]
+        return self._issue_from_pagure_dict(new_issue)
+
+    def issue_close(self, issue_id: int) -> Issue:
+        payload = {"status": "Closed"}
+        issue = self.get_issue_info(issue_id)
```

Comment:
Do we need this call?

Reply:
Oh, I see that you would like to return it and we cannot get info from the response...
(We should create an issue on Pagure for adding issue-info to the response.)

The status of the issue would be outdated in this order -- place this call after the "changing" call.